supplements/extra-life/infernal-machine-rebuild.xml

```diff
@@ -0,0 +1,211 @@
+        </description>
+        <setters>
+            <set name="category">Wondrous Items</set>
+            <set name="cost" currency="gp">0</set>
+            <set name="weight" lb="0">0 lb.</set>
+            <set name="type">Wondrous Item</set>
+            <set name="rarity">Artifact</set>
+            <set name="attunement">true</set>
+        </setters>
+    </element>
+
+<!-- Ruinblade -->
+    <element name="Ruinblade" type="Magic Item" source="Infernal Machine Rebuild" id="ID_WOTC_IMR_MAGIC_ITEM_RUINBLADE">
+        <description>
+            <p><em>Ruinblade</em> is a unique weapon possessed by Moghadam, the architect of the Tomb of Horrors. A symbol of his authority, Moghadam carries the weapon during the time frame of the adventure to intimidate the tomb's workers. (In later years, after Acererak trapped Moghadam in the completed tomb as an undead archwraith, Moghadam was able to use the weapon as his own phylactery.)</p>
+            <p class="indent">A small jade version of the green devil face of the Tomb of Horrors is fashioned into the hilt of the weapon, with the blade extending from its open mouth. The blade always appears ruined and rusting, and causes its surroundings to appear reflected in the blade as if in a state of decay.</p>
+            <p class="indent">You gain a +2 bonus to attack and damage rolls made with this magic weapon. When you hit an undead creature with it, the attack deals an extra 1d8 force damage.</p>
+            <p class="indent">In addition, the weapon has the following properties:</p>
+            <p class="indent"><b><i>Blighted Ruin.</i></b> You can cast the blight spell from the weapon (DC 15). Once used, this property of the weapon can't be used until the next dawn.</p>
+            <p class="indent"><b><i>Destructive Ruin.</i></b> You can cast the disintegrate spell from the weapon, but can target only a nonmagical object or a creation of magical force. Once used, this property of the weapon can't be used until dawn seven days later.</p>
+            <p class="indent"><b><i>Curse.</i></b> This weapon is cursed, and becoming attuned to it extends the curse to you. Until the curse is broken with a remove curse spell or similar magic, you are unwilling to part with the weapon.</p>
+            <p class="indent">The weapon must take direct part in the destruction of 1,000 gp worth of materials and objects every seven days, or a conflict arises between you and the weapon at the end of the seventh day.</p>
+            <p class="indent"><b><i>Sentience.</i></b> <em>Ruinblade</em> is a sentient lawful evil weapon with an Intelligence of 15, a Wisdom of 8, and a Charisma of 16. It has hearing and darkvision out to a range of 120 feet.</p>
```

Comment:
tags that do the same thing (like italics) when separated by only a space can cause issues, so i would try to avoid that.
```suggestion
            <p class="indent"><i><b>Sentience.</b> Ruinblade</i> is a sentient lawful evil weapon with an Intelligence of 15, a Wisdom of 8, and a Charisma of 16. It has hearing and darkvision out to a range of 120 feet.</p>
```